Code/GraphMol/ChemReactions/catch_tests.cpp

```diff
@@ -1198,5 +1198,634 @@ TEST_CASE("V3K rxn blocks") {
     CHECK(rxn->getNumReactantTemplates()==rxn2->getNumReactantTemplates());
     CHECK(rxn->getNumProductTemplates()==rxn2->getNumProductTemplates());   
   }
-
-}
\ No newline at end of file
+ }
+TEST_CASE("CDXML Parser") {
+  SECTION("CDXML REACTION") {
```

Comment:
please move the (very verbose) CDXML data to files instead of embedding it in the tests.